llvm/utils/llvm-lit/CMakeLists.txt

```diff
@@ -35,11 +35,26 @@ if(NOT "${CMAKE_CFG_INTDIR}" STREQUAL ".")
       llvm-lit.in
       ${bi}
       )
+    if (WIN32)
```

Comment:
Have you tested this part?

If it helps, I think it's the same configuration style as https://github.com/llvm/llvm-project/issues/152976#issuecomment-3179880131.